src/renamer.fs

```diff
@@ -206,6 +206,8 @@ module private RenamerImpl =
                 if isTopLevel && (ext || options.hlsl || options.preserveAllGlobals) then
                     if options.preserveExternals then
                         dontRename env decl.name, decl.name
+                    elif env.varRenames.ContainsKey(decl.name) then
+                        env, env.varRenames.TryFind(decl.name) |> Option.get
                     else
                         let env, newName = env.newName env decl.name
                         export env "" decl.name newName
```

Comment:
use match to deconstruct env.varRenames.TryFind(decl.name) to avoid ContainsKey and Option.get?